Control/Parallel/Strategies.hs

```diff
@@ -580,13 +581,36 @@ parListNth n strat = evalListNth n (rparWith strat)
 -- 'parList'
 --
 parListChunk :: Int -> Strategy a -> Strategy [a]
-parListChunk n strat xs
-  | n <= 1    = parList strat xs
-  | otherwise = concat `fmap` parList (evalList strat) (chunk n xs)
+parListChunk = parListChunk'
 
-chunk :: Int -> [a] -> [[a]]
-chunk _ [] = []
-chunk n xs = as : chunk n bs where (as,bs) = splitAt n xs
+parListChunk' :: Int -> (a -> Eval b) -> [a] -> Eval [b]
```

Comment:
There's no point in defining a `parListChunk'`, just define `parListChunk` directly.